scipy/special/_multimethods.py

```diff
@@ -251,6 +251,7 @@
     'obl_rad2',
     'obl_rad2_cv',
     'owens_t',
+    'p_roots',
```

Comment:
This was an alias that was being testing, and as an alias the autogenerated code missed this. There will be some other aliases that need checking.